activiti-core/activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SimpleUserTaskConverterTest.java

```diff
@@ -0,0 +1,51 @@
+package org.activiti.editor.language.xml;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.activiti.bpmn.converter.BpmnXMLConverter;
+import org.activiti.bpmn.model.BpmnModel;
+import org.activiti.bpmn.model.FlowElement;
+import org.activiti.bpmn.model.UserTask;
+import org.junit.Test;
+
+public class SimpleUserTaskConverterTest extends AbstractConverterTest {
+
+    @Test
+    public void convertXMLToModel() throws Exception {
+        BpmnModel bpmnModel = readXMLFile();
+        validateModel(bpmnModel);
+    }
+
+    @Test
+    public void convertModelToXML() throws Exception {
+        BpmnModel bpmnModel = readXMLFile();
+        BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
+        validateModel(parsedModel);
+        deployProcess(parsedModel);
```

Comment:
You can remove `deployProcess(parsedModel);`

Reply:
removed